readthedocs-theme/templates/readthedocs/includes/announcements.html

```diff
@@ -1,51 +1,53 @@
-
-  <section class="ui vertical segment" id="announcements">
-
-    <div class="ui container">
-
-      <div class="ui raised segment">
-        <div class="ui grid stackable three column">
-
-          <div class="column">
-            <div class="ui basic segment">
-              <div><span class="ui label red">Announcement</span></div>
-              <h4>Build errors with docutils 0.18</h4>
-              <p>
-                The latest Docutils and Sphinx updates have been the source of several problems on our builds. If you are experiencing problems related with this read the blog post. Well point you to several solutions.
-              </p>
-              <p>
-                <a href="https://blog.readthedocs.com/build-errors-docutils-0-18/">Learn how to solve it</a>
-              </p>
-            </div>
-          </div>
-          <div class="column">
-            <div class="ui basic segment">
-              <div><span class="ui label primary">News</span></div>
-              <h4>Our Sphinx theme version 1.0.0 is out</h4>
-              <p>
-                We have released a major version 1.0.0 of our Sphinx theme. This adds support for recent versions of Sphinx and docutils among other very cool things. Check out the blog post.
-              </p>
-              <p>
-                <a href="https://blog.readthedocs.com/newsletter-september-2021/">Read more</a>
-              </p>
+<section class="ui vertical segment" id="announcements">
+  <div class="ui container">
+    <div class="ui raised segment">
+      <div class="ui grid stackable three column">
+        
+        {% for article in articles if 'homepage' in article.tags %}
+          {% if loop.index <= 3 %}
+            <div class="column">
+              <div class="ui basic segment">
+                <div>
+                  <span class="ui label {% if article.category == 'Announcement' %}red{% else %}primary{% endif %}">
+                    {# article.category defaults to 'posts' when no category is provided #}
+                    {% if article.category == 'posts' %}
+                      Article
+                    {% else %}
+                      {{ article.category }}
+                    {% endif %}
+                  </span>
+                </div>
+                <h4>{{ article.title }}</h4>
```

Comment:
Probably should link the header to the article as well, users might expect the title to link to the blog post.

Reply:
```suggestion
                <h4><a href="/{{ article.url }}">{{ article.title }}</a></h4>
```